frontend/custom_version_strategy.rst

```diff
@@ -0,0 +1,202 @@
+.. index::
+   single: Asset; Custom Version Strategy
+
+How to Use a Custom Version Strategy for Assets
+===============================================
+
+.. versionadded:: 2.7
+    The Asset component was introduced in Symfony 2.7.
+
+Asset versioning is a technique that improves the performance of web
+applications by adding a version identifier to the URL of your static assets
+(CSS, JavaScript, images, etc.) When the content of the asset changes, the
+identifier changes and the browser is forced to download it again instead of
+using the cached version.
+
+Symfony supports the basic asset versioning thanks to the
+:ref:`version <reference-framework-assets-version>` and
+:ref:`version_format <reference-framework-assets-version-format>` configuration
+options. If your application requires a more advanced versioning, you can create
+your own version strategy.
```

Comment:
Shouldn't we add an example here, what "requires a more advanced versioning"  means?